gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.harness.ts

```diff
@@ -40,6 +40,10 @@ export class ApiLoggingHarness extends ComponentHarness {
 
   private getSamplingTemporalDefaultField = this.locatorFor(MatFormFieldHarness.with({ selector: '[data-testid=temporal_default]' }));
   private getSamplingTemporalLimitField = this.locatorFor(MatFormFieldHarness.with({ selector: '[data-testid=temporal_limit]' }));
+  private getSamplingWindowedCountDefaultField = this.locatorFor(
+    MatFormFieldHarness.with({ selector: '[data-testid=windowedCount_default]' }),
+  );
+  private getSamplingWindowedCountLimitField = this.locatorFor(MatFormFieldHarness.with({ selector: '[data-testid=windowedCount_limit]' }));
```

Comment:
"Member 'getSamplingWindowedCountDefaultField' is never reassigned; mark it as `readonly`."
```suggestion
  private readonly getSamplingWindowedCountLimitField = this.locatorFor(MatFormFieldHarness.with({ selector: '[data-testid=windowedCount_limit]' }));
```